scipy/optimize/zeros.py

```diff
@@ -843,6 +847,15 @@ def brenth(f, a, b, args=(),
         Object containing information about the convergence. In particular,
         ``r.converged`` is True if the routine converged.
 
+    References
+    ----------
+    .. [BusAndDekker1975]
+       Bus, J. C. P., Dekker, T. J.,
+       *Two Efficient Algorithms with Guaranteed Convergence for Finding a Zero
+       of a Function*. ACM Transactions on Mathematical Software, Vol. 1, Issue
+       4, Dec. 1975, pp. 330-345. Section 3: "Algorithm M".
+       DOI:https://doi.org/10.1145/355656.355659
```

Comment:
I suppose since @charris is the author we could ask them for quick look at the docs changes here.

Seems like an improvement to a non-expert like me. I suppose the DOI might be made into a link.

Reply:
I fixed this by using `:doi:` reST directive.